pkg/blob/controllerserver.go

```diff
@@ -679,6 +700,69 @@ func (d *Driver) DeleteBlobContainer(ctx context.Context, subsID, resourceGroupN
 	})
 }
 
+// CopyBlobContainer copies a blob container in the same storage account
+func (d *Driver) CopyBlobContainer(ctx context.Context, req *csi.CreateVolumeRequest, accountKey, dstContainerName, storageEndpointSuffix string, secrets map[string]string) error {
```

Comment:
if `secrets` is not needed in this func, remove this parameter

Reply:
done